src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -42,7 +42,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.1.1" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.174-preview" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14-11656" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="3.0.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="3.0.4" />
```

Comment:
Should this be a separate PR. Added to this for the sake of build times.

Reply:
Fine here.